tests/units/Glpi/RichText/RichText.php

```diff
@@ -397,6 +397,11 @@ public function () {
             'encode_output_entities' => false,
             'expected_result' => '<span data-form-tag="true">Tag label</span>',
         ];
+        yield 'Do not remove data-form-tag-value property' => [
+            'content' => '<span data-form-tag-value="my-value">Tag label</span>',
+            'encode_output_entities' => false,
+            'expected_result' => '<span data-form-tag-value="my-value">Tag label</span>',
+        ];
```

Comment:
You could merge it and the 2 previous yield into a unique case named `Forms tags must be preserved`.

Reply:
I hesitated between one test with all properties and multiple tests that each check one property.

I went for this version because I guess it could be slightly easier to debug in case one test start to fails (comparing long strings in failing tests can be a bit hard sometimes (more properties will probably be added in the futures PR)).

I'll let you decide, let me know if you prefer one single test.

Reply:
You can keep it like that.